### ISSUE TITLE
1.15 

### ISSUE THREAD

**Issue by matthewjessee16** [GlobbyPotato/Rockhounding_Chemistry#118]
Will this mod come to 1.15

**Comment by matthewjessee16** [comment]
Can someone plz answer me

**Comment by Shazuli** [comment]
Hi @matthewjessee16.
I'm not part of the dev team, but I have a slow conversation with him in [my](96) closed issue:

**tl;dr** he says he says he would like to update past 1.12 but real life has his focus (most likely studies), but as soon as some free time and motivation come back he would most likely start converting the project past 1.12.
Something to keep in mind is that 1.16 is just around the corner and doing a Minecraft version update of a project is always going to be a pain, so it will most likely go straight to 1.16 when Forge is available.

Then there's the dependencies: One is [Guidebook](https://www.curseforge.com/minecraft/mc-mods/guidebook), which is definitely needed. [Bookshelf](https://www.curseforge.com/minecraft/mc-mods/bookshelf) is more of a convenient dependency and he says would like to remove from the project. Lastly, there's [Redstone Flux](https://www.curseforge.com/minecraft/mc-mods/redstone-flux) which is still stuck in 1.12. That one is a bit trickier as that is what the mod uses to handle power, but a custom energy interface used only in the mod with Forge's energy API to handle mod compatibility would resolve that issue and improve performance. But as you might have already figured out, easier said than done.

The whole project is most likely needed to be built from the ground up again, Forge basically did that for their whole project when updating to 1.13. And as a mod developer, I know the pain of doing such a thing. So remember to be patient, life always goes first.

Hope you got the answer you wanted.

**Comment by matthewjessee16** [comment]
I did thx @Shazuli
